### PR TITLE
Do not start connsCleaner on SetConnectionClosed requests.

### PR DESCRIPTION
**Pull request by moredure**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -1459,7 +1459,7 @@ func (c *HostClient) acquireConn(reqTimeout time.Duration) (cc *clientConn, err
 		if c.connsCount < maxConns {
 			c.connsCount++
 			createConn = true
-			if !c.connsCleanerRun {
+			if !(c.connsCleanerRun || connectionClose) {
```
Comment (erikdubbelboer) [inline client.go:1462]:
```suggestion
			if !c.connsCleanerRun && !connectionClose {
```

**Review by erikdubbelboer — CHANGES_REQUESTED** [review]
Good optimization, just one small change request to make the condition a bit cleaner.

File: client.go (hunk 1 above)
Comment (moredure) [reply client.go:1462]: Just de morgana rule) I think readability depends on reader in this case) But ok, it's not a problem.

**Comment by erikdubbelboer** [comment]
Thanks!

**Comment by moredure** [comment]
Thank you too!

**Pull request merged**